src/frontend/src/lib/components/hero/Header.svelte

```diff
@@ -14,8 +14,17 @@
 
 	let back = false;
 	$: back = isSubRoute($page);
+
+	let hideName = false;
+	$: $authSignedIn, updateHideName();
+
+	const updateHideName = () => {
+		hideName = window.innerWidth < 768 && $authSignedIn;
```

Comment:
Let's avoid the use of JavaScript and hardcoded size for display purpose. Please resolve this with CSS.